src/decision/src/VisionDecision.cpp

```diff
@@ -1,15 +1,21 @@
 /*
- * Created By: Gareth Ellis
+ * Created By: Robyn Castro
  * Created On: September 22, 2016
  * Description: The vision decision node, takes in an image from the robot's
  *              camera and produces a recommended twist message
  */
-
 #include <VisionDecision.h>
+#include <tiff.h>
+
+#define PI 3.14159265
+#define null -1
+#define INVALID 91
```

Comment:
Not sure what `INVALID` refers to? After looking at the code I get that it's something to do with the turn angle? Name should be a bit more specific